commit 7e2c4ca4d6caf0a832ebd59e645659ca26ac7ea6 (refs/changes/01/1701/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-05-04T20:15:39+00:00 (2 years, 5 months ago)
    
    NAPI-401 VL2 and VL3 events should be inserted with null etags
    Reviewed by: Jason King <jason.king@joyent.com>
    Approved by: Jason King <jason.king@joyent.com>

diff --git a/lib/consumer.js b/lib/consumer.js
index 99e38cc..84695be 100644
--- a/lib/consumer.js
+++ b/lib/consumer.js
@@ -317,8 +317,7 @@ function logReq(opts, callback) {
      * limit as the _maximum bytes_ to return; even should varpd requests bytes
      * that would total >1000 messages, we can safely return the first 1000.
      */
-    var req = opts.moray.findObjects(NAMES.net_events, filter, searchOpts,
-        true);
+    var req = opts.moray.findObjects(NAMES.net_events, filter, searchOpts);
 
     req.once('error', function searchError(err) {
         opts.log.error(err, 'logreq search error');
diff --git a/lib/producer.js b/lib/producer.js
index 4fd9818..f59f406 100644
--- a/lib/producer.js
+++ b/lib/producer.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2016, Joyent, Inc.
+ * Copyright 2017, Joyent, Inc.
  */
 
 /*
@@ -89,10 +89,7 @@ function overlayMappingBatch(opts) {
         bucket: NAMES.mac_ip,
         key: rec.key,
         operation: 'put',
-        value: rec.value,
-        options: {
-            // XXX: set etag?
-        }
+        value: rec.value
     };
 }
 
@@ -186,9 +183,6 @@ function underlayMappingBatch(opts) {
             cn_uuid: opts.cn_uuid,
             ip: opts.ip,
             port: opts.port
-        },
-        options: {
-            // XXX: set etag?
         }
     };
 }
@@ -270,6 +264,9 @@ function vl2CnEventBatch(opts) {
             bucket: NAMES.net_events,
             key: uuid,
             operation: 'put',
+            options: {
+                etag: null
+            },
             value: {
                 cn_uuid: cn,
                 vnet_id: opts.vnet_id,
@@ -308,6 +305,9 @@ function vl3CnEventBatch(opts) {
             bucket: NAMES.net_events,
             key: uuid,
             operation: 'put',
+            options: {
+                etag: null
+            },
             value: {
                 cn_uuid: cn,
                 vnet_id: opts.vnet_id,
